deployment/migrations/scripts/0002-refresh-chain-pins.py

```diff
@@ -0,0 +1,42 @@
+"""
+This migration resets the chain height in order to ensure that a PermanentPin is present for every chaindata
+committed on a chain. It has to be run after the introduction of PermanentPin in version 0.2.0.
+"""
+
+import asyncio
+import logging
+from typing import Optional
+
+from configmanager import Config
+
+from aleph.config import get_defaults
+from aleph.model import PermanentPin, init_db
+from aleph.model.chains import Chain
+
+logger = logging.getLogger()
+
+
+async def async_upgrade(config_file: Optional[str], **kwargs):
+    config = Config(schema=get_defaults())
+    if config_file is not None:
+        config.yaml.load(config_file)
+
+    init_db(config=config, ensure_indexes=False)
+
+    if (await PermanentPin.count(filter={})) < 5000:
```

Comment:
Maybe use a constant here to be more explicit? I think we should document why we use 5000 here.